utils/src/main/scala/za/co/absa/enceladus/utils/general/ProjectMetadata.scala

```diff
@@ -15,8 +15,10 @@
 
 package za.co.absa.enceladus.utils.general
 
-object ProjectMetadata {
-
-  val enceladusVersion: String = getClass.getPackage.getImplementationVersion
+trait ProjectMetadata {
 
+  def enceladusVersion(): String = {
```

Comment:
**For consideration:** It's utils, kind of shared module. What about calling it `projectVersion`
(No a change request, it's really just an idea. 😉 )

Reply:
I agree it's a better name